### PR TITLE
Multiple fixes for newer RN versions

### PR DESCRIPTION
**Pull request by TriPSs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
   },
   "peerDependencies": {
     "react": "*",
-    "react-native": "*"
+    "react-native": "*",
+    "react-native-webview": "*"
```
Comment (tijs) [inline package.json:14]: @TriPSs any particular reason your adding this as a peer dependency instead of a direct dependency?

Comment (TriPSs) [reply package.json:14]: I think this was to not force a specific version to the users and to keep inline with react native, as that one was also in the peerDependencies.

**Comment by tijs** [comment]
Ah yeah that makes sense. So as long as react-native keeps including it it’s good to go.

**Comment by Anwardo** [comment]
Thanks for the input guys, this looks good. Expect a new release with these changes next week.

**Pull request merged**